var/spack/repos/builtin/packages/dislin/package.py

```diff
@@ -33,11 +33,48 @@ class Dislin(Package):
     homepage = "http://www.mps.mpg.de/dislin"
     url      = "ftp://ftp.gwdg.de/pub/grafik/dislin/linux/i586_64/dislin-11.0.linux.i586_64.tar.gz"
 
+    version('11.1.linux.i586_64', '34218c257efedaf706f058bdf111ce9d')
     version('11.0.linux.i586_64', '6fb099b54f41db009cafc702eebb5bc6')
 
     depends_on('motif')
     depends_on('mesa')
 
+    provides('dislin_d')
+    provides('discpp')
+    provides('disjava')
+    provides('dislnc')
+    provides('dislnc_d')
```

Comment:
Is there anything that depends on all of these virtual packages? Are there any other packages that provide the same virtual packages?